src/content/docs/es/guides/bursa/007-address-commands.md

```
@@ -0,0 +1,66 @@
+
+> **Bursa soporta todos los tipos de direcciones CIP-0019:**
+> - Direcciones base (credenciales de pago + stake)
+> - Direcciones enterprise (solo pago)
+> - Direcciones pointer (pago + puntero de stake)
+> - Direcciones de recompensa (solo stake)
+> - Direcciones Byron/Bootstrap (legado)
+
+#### Construcción de dirección
+- Direcciones base (credenciales de pago + stake)
+```
+./bursa address build --payment-key addr_vk1... --stake-key stake_vk1... --network mainnet
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)